python/pip_install/pip_repository.bzl

```diff
@@ -178,6 +180,9 @@ def _pip_repository_impl(rctx):
     if result.return_code:
         fail("rules_python failed: %s (%s)" % (result.stdout, result.stderr))
 
+    # We need a BUILD file to load the generated requirements.bzl
+    rctx.file("BUILD.bazel", _BUILD_FILE_CONTENTS.format(" ".join([str(a) for a in args])))
```

Comment:
sure, my thinking here was just that there's no harm documenting the command that was used to generate a requirements.bzl for non-incremental as well, it will just be the extract_wheels command line

Reply:
Yeah feel free to ignore. Just thought it might be confusing / unneccesary.